.circleci/config.yml

```diff
@@ -257,6 +257,27 @@ jobs:
               --junitxml=test-reports/junit.xml \
               --server http://localhost:8000
 
+  containerized-tests:
+    machine:
+      docker_layer_caching: true
+    steps:
+      - run:
+          name: Install Docker ComposeCreate virtualenv
+          command: |
+            set -x
+            sudo bash -c "curl -L https://github.com/docker/compose/releases/download/1.11.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose"
+            sudo chmod +x /usr/local/bin/docker-compose
```

Comment:
Is sudo needed here? We should already be running as root.

Reply:
Fixed